ui/apps/dashboard/src/components/Insights/InsightsSQLEditor/constants.ts

```diff
@@ -0,0 +1,41 @@
+import type { SQLCompletionConfig } from '@inngest/components/SQLEditor/types';
+
+const KEYWORDS = [
+  'AND',
+  'AS',
+  'ASC',
+  'BETWEEN',
+  'DESC',
+  'DISTINCT',
+  'FALSE',
+  'FROM',
+  'GROUP BY',
+  'IS',
+  'LIKE',
+  'LIMIT',
+  'NOT',
+  'NULL',
+  'OFFSET',
+  'OR',
+  'ORDER BY',
+  'SELECT',
+  'TRUE',
+  'WHERE',
+] as const;
+
+const FUNCTIONS = [
+  { name: 'AVG', signature: 'AVG(${1:column})' },
+  { name: 'COUNT', signature: 'COUNT(${1:column})' },
+  { name: 'MAX', signature: 'MAX(${1:column})' },
+  { name: 'MIN', signature: 'MIN(${1:column})' },
+  { name: 'SUM', signature: 'SUM(${1:column})' },
```

Comment:
what does `${1:column}` mean? does it mean the editor assumes one argument?

in the future we can try to generate this list from here https://github.com/inngest/monorepo/blob/develop/pkg/insights/allowed_clickhouse_functions.go

Reply:
oh I see, it puts `column` as a placeholder to the function?